docs/manifest.jsonc

```diff
@@ -32,7 +32,14 @@
       }
     },
     "chrome_url_overrides": {},
-    "commands": {},
+    "commands": {
+          "_execute_browser_action": {
+            "suggested_key": {
+              "default": "Ctrl+Shift+Y",
```

Comment:
any reason why you chose this shortcut over something shorter like `shift + 2`? Maybe we can just use `cmd + shift + A` (A for analogue, and since the keys are closer together than Y)

Reply:
`cmd + shift + A` makes way more sense. Can't do `shift + 2` because the shortcut needs either `cmd` or `alt` (but not both).

Reply:
Ahh, cool that makes sense. Yea let's run with `cmd + shift + A` then!